content/article/what-s-new-on-cpan---january-2024.md

```diff
@@ -0,0 +1,118 @@
+{
+   "title" : "What's new on CPAN - January 2024",
+   "authors" : [
+      "mathew-korica"
+   ],
+   "description" : "A curated look at January's new CPAN uploads",
+   "draft" : true,
```

Comment:
I think we can just set `"draft": false` for future PRs, as I'll need to change it on order to publish anyway.